sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityAsyncTests.java

```diff
@@ -374,7 +404,7 @@ public void revokeTokenWithResponseWithNullUser() {
 
 
     @Test
-    public void getToken() {
+    public void getTokenWithChatScope() {
```

Comment:
The same comment that I left for `createUserAndTokenWithChatScope`

Reply:
Done